exts/tex.py

```diff
@@ -0,0 +1,67 @@
+"""
+TeX extension for qrm
+---
+Copyright (C) 2021 thxo
+
+This file is part of qrm2 and is released under the terms of
+the GNU General Public License, version 2.
+"""
+
+import aiohttp
+from io import BytesIO
+from urllib.parse import urljoin
+
+import discord
+import discord.ext.commands as commands
+
+import common as cmn
+
+
+class TexCog(commands.Cog):
+    # TeX rendering performed remotely using the rTeX project.
+    # Documentation: https://rtex.probablyaweb.site/docs
+    RTEX_INSTANCE = "https://rtex.probablyaweb.site/"
```

Comment:
to think about: this maybe could be a config variable. I run my own rtex for my qsl generator (because I want more packages than it has), and someone else running the bot could also want the same.